common/components/common/groups/AboutGroupDisplay.jsx

```diff
@@ -320,20 +159,26 @@ class AboutGroupDisplay extends React.PureComponent<Props, State> {
       <IconLinkDisplay key={i} link={link}/>
     );
   }
-
-  _renderPositions(): ?Array<React$Node> {
-    const group: GroupDetailsAPIData = this.state.group;
-    const canApply: boolean = CurrentUser.canVolunteerForGroup(group);
-    return group && group.group_positions && _.chain(group.group_positions).sortBy(['roleTag.subcategory', 'roleTag.display_name']).value()
-      .map((position, i) => {
-        return <AboutPositionEntry
-          key={i}
-          group={group}
-          position={position}
-          onClickApply={canApply ? this.handleShowVolunteerModal.bind(this, position) : null}
-        />;
-      });
-    }
+  
+  _renderProjectList(): ?$React$Node {
+    return (
+      <React.Fragment>
+        {this.state.group.group_projects.map(
+          (project, index) => {
+            return (
+              <div className="col-sm-12 col-lg-6">
+                <ProjectCard
+                  project={ProjectAPIUtils.projectFromAPIData(project)}
```

Comment:
Please add a class here, something like `AboutGroup-card` because we're going to have to write some CSS rules to prevent all the card text from being hyperlink orange due to the overriding `a` rules on this page.

Reply:
Solved this a different way after writing the review, so I'm marking resolved.